wrapper/src/main/java/software/amazon/jdbc/hostavailability/HostAvailabilityStrategy.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package software.amazon.jdbc.hostavailability;
+
+import software.amazon.jdbc.AwsWrapperProperty;
+
+public interface HostAvailabilityStrategy {
+
+  static final AwsWrapperProperty HOST_AVAILABILITY_STRATEGY_MAX_RETRIES = new AwsWrapperProperty(
```

Comment:
Static properties in interface declaration is not a best practise. Need to relocate them somewhere else.

Reply:
Not sure if it was the best place. but moved it to `HostAvailabilityStrategyFactory.java`